### PR TITLE
Add update fixes to 1.0.x

### PR DESCRIPTION
**Pull request by IsankaSR**
### Purpose
Adding update fixes.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -630,18 +640,19 @@
         <carbon.commons.version>4.7.22</carbon.commons.version>
         <carbon.commons.imp.pkg.version.range>[4.7.0, 5.0.0)</carbon.commons.imp.pkg.version.range>
         <carbon.logging.version>4.4.39</carbon.logging.version>
+        <totp.authenticator.version>2.2.10</totp.authenticator.version>
```
Comment (IsankaSR) [inline pom.xml:643]: Need to update proper totp authenticator version

File: pom.xml [hunk 2]
```diff
@@ -630,18 +640,19 @@
...
 
         <carbon.base.imp.pkg.version.range>[1.0.0, 2.0.0)</carbon.base.imp.pkg.version.range>
         <org.wso2.carbon.identity.outbound.auth.oidc>5.1.20</org.wso2.carbon.identity.outbound.auth.oidc>
 
         <carbon.extension.identity.authenticator.version>3.0.0</carbon.extension.identity.authenticator.version>
         <org.wso2.carbon.identity.association.account>5.1.5</org.wso2.carbon.identity.association.account>
         <identity.extension.utils>1.0.8</identity.extension.utils>
-        <carbon.identity.framework.version>5.18.176</carbon.identity.framework.version>
+        <carbon.identity.framework.version>5.18.187.32</carbon.identity.framework.version>
```
Comment (IsankaSR) [inline pom.xml:651]: Need to update proper framework version

File: pom.xml [hunk 3]
```diff
@@ -630,18 +640,19 @@
...
         <carbon.identity.framework.imp.pkg.version.range>[5.0.0, 6.0.0)</carbon.identity.framework.imp.pkg.version.range>
-        <carbon.identity.oauth.version>6.4.98</carbon.identity.oauth.version>
+        <carbon.identity.oauth.version>6.4.111.11</carbon.identity.oauth.version>
```
Comment (IsankaSR) [inline pom.xml:653]: Need to update proper version

File: pom.xml [hunk 4]
```diff
@@ -630,18 +640,19 @@
...
         <carbon.identity.oauth.imp.pkg.version.range>[6.1.0, 7.0.0)</carbon.identity.oauth.imp.pkg.version.range>
-        <identity.governance.version>1.4.16</identity.governance.version>
+        <identity.governance.version>1.4.72.17</identity.governance.version>
```
Comment (IsankaSR) [inline pom.xml:655]: Need to update proper governance version

File: pom.xml (hunk 2 above)
Comment (IsankaSR) [reply pom.xml]: Updated with eacaf45

File: pom.xml (hunk 3 above)
Comment (IsankaSR) [reply pom.xml]: Updated with eacaf45

File: pom.xml (hunk 4 above)
Comment (IsankaSR) [reply pom.xml]: Updated with eacaf45

**Review by nilasini — APPROVED** [review]

**Pull request merged**